pkg/services/accesscontrol/filter.go

```diff
@@ -26,37 +25,49 @@ var (
 	allowAllQuery = SQLFilter{" 1 = 1", nil}
 )
 
+type SQLFilter struct {
+	Where string
+	Args  []interface{}
+}
+
 // Filter creates a where clause to restrict the view of a query based on a users permissions
 // Scopes for a certain action will be compared against prefix:id:sqlID where prefix is the scope prefix and sqlID
 // is the id to generate scope from e.g. user.id
-func Filter(ctx context.Context, sqlID, prefix, action string, user *models.SignedInUser) (SQLFilter, error) {
+func Filter(user *models.SignedInUser, sqlID, prefix string, actions ...string) (SQLFilter, error) {
```

Comment:
Please update the comment for the function. I think it is worth describing that in order to include the resource in the result, the user must have permission for all actions in the scope that includes the resource (examples would be nice as well)